src/tools/pkcs11-tool.c

```diff
@@ -5630,6 +5680,7 @@ static int read_object(CK_SESSION_HANDLE session)
 					util_fatal("cannot parse EC_PARAMS");
 				EVP_PKEY_assign_EC_KEY(pkey, ec);
 #else
+/* TODO needs debugging */
```

Comment:
does this mean that it does not work now or something else? The comment is not much self-explanatory.

Reply:
It was meant that I did not test this and `pkcs11-tool --test` does not test ECDSA.  I will  try `pkcs11-tool -sign` with an ec key and verify the signature.